packages/fast-components-class-name-contracts-msft/src/action-toggle/index.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * The class name contract for the action toggle component
+ */
+export interface ActionToggleClassNameContract {
+    /**
+     * The root of the action toggle component
+     */
+    actionToggle: string;
```

Comment:
I think this should be optional

Reply:
yes, 100%. Technically all of these can still exist as *unstyled* components per our pattern

Reply:
I got the opposite feedback on the previous Action Trigger PR on the same issue:
![image](https://user-images.githubusercontent.com/7649425/49830612-912d4000-fd46-11e8-8f66-8a4222c272db.png)